projects/assets-library/assets/styles/_color-palette.scss

```diff
@@ -84,6 +84,8 @@ $cloud-3: #a1c5e0;
 
 $brown-1: #9e4c41;
 
+$grey-200: #e5e6e6;
```

Comment:
we no longer use the colors with suffix 100, 200 etc. That used to be the case earlier. We converted all such values into a correspoding gray-1, 2 ... n pattern when we redesigned. This might be a remnant from the earlier design system. All the old gray-200 values are now gray-1 in the system. You can either use that if its suitable or consult with @michael-traceable to get a suitable color for this.

Reply:
Currently changed this to `$grey-1`

Reply:
I meant we should use one of the existing colors from the color palette. A `$gray-1` already exists.

Reply:
Blargh... If it's used... it should be `gray-2`... if it's not used, then just remove it.

Reply:
Yeah, made the changes